change title on README

natafesenko opened the pull request:
Rename a Game

hectorsector committed:
change title on README

natalinatusik commented on README.md:
```diff
@@ -1,4 +1,4 @@
+## Welcome to awesome game of Tetris
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
```
```suggestion
You can play the game at: https://natalinatusik.github.io/github-games/
```

natafesenko committed:
Update README.md

```suggestion
You can play the game at: https://natalinatusik.github.io/github-games/
```

Co-authored-by: github-learning-lab[bot] <37936606+github-learning-lab[bot]@users.noreply.github.com>
